.github/workflows/pre-release.yml

```diff
@@ -2,9 +2,6 @@ name: Pre-release checks
 
 on:
   workflow_dispatch:
```

Comment:
Keeping this file around as it might prove useful for other pre-release types of checks in the future and it can be run manually from the Actions screen.